src/BizHawk.Client.EmuHawk/FormBase.cs

```diff
@@ -94,5 +94,40 @@ public void UpdateWindowTitle()
 			=> base.Text = Config?.UseStaticWindowTitles == true
 				? (_windowTitleStatic ??= WindowTitleStatic)
 				: WindowTitle;
+
+		// Alt key hacks. We need this in order for hotkey bindings with alt to work.
+		/// <summary>sends a simulation of a plain alt key keystroke</summary>
+		internal void SendPlainAltKey(int lparam)
```

Comment:
These should be `public`.

Reply:
`FormBase` and the caller, `Input`, aren't semantically related. They may end up in separate projects in the future.

Reply:
The caller is `MainForm`.